src/main/views/layouts/default.layout.njk

```diff
@@ -0,0 +1,49 @@
+ 
+{% from "govuk/components/header/macro.njk" import govukHeader %}
+{% from "govuk/components/phase-banner/macro.njk" import govukPhaseBanner %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+
+{% from "../macros/page-heading/page-heading.njk" import pageHeading %}
+{% from "../macros/alert/alert.njk" import alert %}
+{% from "../macros/task-list/task-list.njk" import taskList %}
+{% from "../macros/task-list/sub-task-list.njk" import subTaskList %}
```

Comment:
Why we have got those taskList in. the default layouot? Shouldn't they be imported in view specific teempaltes? (sorry if this is best practise, but seems a bit weird!)

Reply:
I like having the default.layout import all components we will need. It means we have consistency one place to look for macros.

Reply:
Ok cool, but then you reimported that macros in the task-list view anyway so I was confused. I don't have strong opinion if other devs like it, I am happy with it.

Reply:
task-list.njk must be outdated on github. We do not import further macros on the template.

Reply:
not sure i understand why taskList macro's are imported in default.layout! I have pulled latest. You may have some local changes which i don't see.